scenarios/cutin/scenario.py

```diff
@@ -0,0 +1,41 @@
+import os
```

Comment:
I could suggest putting all the BM scenarios in a subfolder like "scenarios/behavior_models/<task>".